### ISSUE TITLE
Cloud Lab Quo Vadis

### ISSUE THREAD

**Issue by culmat** [baloise/CICD-pipeline#43]
**Context**

- Current Cloud Lab funding runs until end Feb
- As of current planning the private Cloud landing zone delivery will take place mid March earliest
- We do net yet know how to transfer from the Cloud Lab to the private Cloud landing zone
- The scope of functionality that we have implemented in the Cloud Lab is relatively narrow
- Lately mainly the SIAM team has used the Lab Environment for MyCloud API exploration

**Questions**

Who wants to achieve what in the remaining time until end of Feb?
Who is in favour of extending the Cloud Lab and for what reasons?

**Comment by culmat** [comment]
My personal / BizDevOps guild view on this.

1.  The Cloud Lab was a precious learning environment that enabled us to do ground work to prepare CI / CD capabilities.
2. We have postponed onboarding of customers teams to cloud lab functionality from our OKR to Q2 in favour of kicking off the community of practice DevOps. So we do not have a use case for the cloud lab from the guild any more.
3. Because of the limited functional scope and the unsure migration path to the private Cloud landing zone I doubt that migrating functionality will be an advantage over reimplementing.

I would not extend the Lab and focus efforts on building the private Cloud landing zone.

**Comment by vangual** [comment]
From the point of view of the SIAM Team the MyCloud instance is crucial in our efforts to interface with HCL. Considering the delays in getting any kind of instance running locally we have no other way of developing and testing our calls to HCL.

This however only concerns the following functionality of the lab:
- MyCloud instance to speak to
- Conncectivity to:
  * Mycloud Lab instance
  * Jump Host
  * Connectivity both via web-channels and ssh channels as they exist at the time of writing this.
- Ability to deploy, modify and remove testing-versions of the order items (VMs, Databases, ...)

What is not included in this requirement of SIAM:
- Jenkins
- Nexus

To summarize: unless the local dev instance of MyCloud is up and running before the end of february, the oneit marketplace team absolutely requires at least parts of the lab to still be available. 

@matal0 @Solaflex FYI

**Comment by matal0** [comment]
A fully agree with Sam. Without myCloud in the lab and access via jump server, we cannot continue with the implementation of the MVP. Although the features and the API offered by the lab are different than what we will see in the private cloud, it is the only way for us to simulate our processes (meaning to perform integration tests).
Btw, we are fully aware of the fact that some steps within our process need to be reworked when switching from lab to private cloud.
